stablehlo/integrations/python/mlir/dialects/stablehlo.py

```diff
@@ -17,3 +17,11 @@
 # pylint: disable=wildcard-import,relative-beyond-top-level,g-import-not-at-top
 from ._stablehlo_ops_gen import *
 from .._mlir_libs._stablehlo import *
+
+
+def get_earliest_forward_compatible_version():
+  """Return the earliest StableHLO version that the current StableHLO version
+
+  is still forward compatible with.
+  """
+  return "0.9.0"
```

Comment:
Any thoughts on single-sourcing this number? Our python build confuses me a little bit, we could add this as a method in StablehloModule.cpp which calls Serialization.h.. but how does building / including that file work? Would we be expecting all users of stablehlo python API's to include serialization dependencies in their build dependencies?

Reply:
I think it's a good idea to have a single source of truth. 

The concern about incurring additional dependency on StablehloModule.cpp is a good point, but since we aren't yet guaranteeing much about our public APIs, I think it's fine to have this dependency. We can refactor later if this turns out to be a problem.